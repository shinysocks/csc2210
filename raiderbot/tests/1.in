+--------------------+
|     #              |
|  **          -     |
| #         #        |
|                    |
| O                  |
|    *#      +   #   |
|                    |
|  ####              |
|              ##    |
|                    |
+--------------------+
1 1
ees
