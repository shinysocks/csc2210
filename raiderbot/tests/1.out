Initial state: Robot at 1, 1 (0 gold)
+--------------------+
|     #              |
| R**          -     |
| #         #        |
|                    |
| O                  |
|    *#      +   #   |
|                    |
|  ####              |
|              ##    |
|                    |
+--------------------+
========================================
Executing ees
Robot at 3, 2 (2 gold)
+--------------------+
|     #              |
|              -     |
| # R       #        |
|                    |
| O                  |
|    *#      +   #   |
|                    |
|  ####              |
|              ##    |
|                    |
+--------------------+
========================================
Robot has completed its task.
